lib/process.js

```diff
@@ -78,5 +98,20 @@ module.exports = function (src, path) {
     }
   }
 
+  if (Array.isArray(parts.styles) && parts.styles.length > 0) {
+    const styleStr = parts.styles.map(ast => {
```

Comment:
we should only convert style partss that acually have a "module" attribute, right?

Otherwise we would create a `$styles` object on a component that onnly has "normal" styles like `<style lang="scss">`.